.snyk

```diff
@@ -0,0 +1,47 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.14.1
+
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  SNYK-DEBIAN10-SYSTEMD-345386:
```

Comment:
I'm not sure the situation will improve significantly if we keep Debian base images. The `openjdk:11-slim` image has the security apt sources commented out:

```
$ docker run -ti openjdk:11 cat /etc/apt/sources.list
# deb http://snapshot.debian.org/archive/debian/20201117T000000Z buster main
deb http://deb.debian.org/debian buster main
# deb http://snapshot.debian.org/archive/debian-security/20201117T000000Z buster/updates main
deb http://security.debian.org/debian-security buster/updates main
# deb http://snapshot.debian.org/archive/debian/20201117T000000Z buster-updates main
deb http://deb.debian.org/debian buster-updates main
```

❓ I think the recommended way to deal with security problems is to update the base image. Shall we do that next?

Reply:
there are no upgrade paths for most of these issues without going to the development release of debian. 

we can look into using different base images, but i couldn't find anything obvious that would be suitable via quick google